ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -108,6 +108,26 @@ tenants:
 +             always_redistribute_igmp: true
 ```
 
+### `vrf_id` is used instead of `vrf_vni` to calculate the IP address of EVPN underlay multicast group for a VRF
+
+AVD versions below 5.0.0 uses `vrf_vni` in the algorithm to set the offset in order to get the IP address of EVPN underlay multicast group.
+
+In AVD version 5.0.0, `vrf_id` is used instead of `vrf_vni` to calculate the offset for IP address of EVPN underlay multicast group for a VRF when `evpn_l3_multicast` is enabled. The configurations will only change in case `vrf_id` and `vrf_vni` are set to different values.
+
+To retain the previous configuration, either exclude the `vrf_id` from the input or ensure that `vrf_id` is identical to `vrf_vni`.
+
+```diff
+tenants:
+  - name: Tenant_C
+    evpn_l3_multicast:
+      enabled: true
+    vrfs:
+      - name: TEN_C_L3_MULTICAST_ENABLED_130_131
+        vrf_vni: 31
++       vrf_id: 31
+        <...>
+```
```

Comment:
This is not a great suggestion, since it will affect many other things. We should either suggest a structured config snip, or provide a knob to set the group directly under the VRF.

Reply:
The knob should be `evpn_l3_multicast.evpn_underlay_l3_multicast_group`.

Reply:
Added knob.